### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Serhii-Leonenko**

### CODE REVIEW

File: cinema/serializers.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(max_length=55)
+    description = serializers.CharField(required=False, max_length=500)
```
Comment (mens0) [inline cinema/serializers.py:9]: i think required=False is set by default

**Review by mens0 — APPROVED** [review]

File: cinema/serializers.py (hunk 1 above)
Comment (Serhii-Leonenko) [reply cinema/serializers.py:9]: No, on the contrary)

File: cinema/serializers.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(max_length=55)
```
Comment (TPolina) [inline cinema/serializers.py:8]: why do you have different max_length values in the model and serializer?

**Review by TPolina — CHANGES_REQUESTED** [review]

File: cinema/serializers.py (hunk 2 above)
Comment (Serhii-Leonenko) [reply cinema/serializers.py:8]: yes, my mistake

**Review by TPolina — APPROVED** [review]

**Pull request closed without merging**